public/app/features/dashboard/components/TransformationsEditor/TransformationsEditor.tsx

```diff
@@ -414,4 +623,133 @@ const getStyles = (theme: GrafanaTheme2) => {
   };
 };
 
+interface TransformationsGridProps {
+  transformations: Array<TransformerRegistryItem<any>>;
+  showIllustrations?: boolean;
+  onClick: (id: string) => void;
+}
+
+function TransformationsGrid({ showIllustrations, transformations, onClick }: TransformationsGridProps) {
+  const styles = useStyles2(getTransformationsGridStyles);
+
+  return (
+    <div className={styles.grid}>
+      {transformations.map((transform) => (
+        <Card
+          key={transform.id}
+          className={styles.card}
+          aria-label={selectors.components.TransformTab.newTransform(transform.name)}
+          onClick={() => onClick(transform.id)}
+        >
+          <Card.Heading className={styles.heading}>
+            <>
+              <span>{transform.name}</span>
+              <span
+                className={css`
+                  margin-left: 5px;
+                `}
+              >
+                <PluginStateInfo className={styles.badge} state={transform.state} />
+              </span>
+            </>
+          </Card.Heading>
+          <Card.Description className={styles.description}>
+            <>
+              <span>{getTransformationsRedesignDescriptions(transform.id)}</span>
+              {showIllustrations && (
+                <span>
+                  <img className={styles.image} src={getImagePath(transform.id)} alt={transform.name} />
+                </span>
+              )}
+            </>
+          </Card.Description>
+        </Card>
+      ))}
+    </div>
+  );
+}
+
+const getTransformationsGridStyles = (theme: GrafanaTheme2) => {
+  return {
+    grid: css`
+      display: grid;
+      grid-template-columns: repeat(auto-fill, minmax(220px, 1fr));
+      grid-auto-rows: 1fr;
+      gap: ${theme.spacing(2)} ${theme.spacing(1)};
+      width: 100%;
+    `,
+    card: css`
+      grid-template-rows: min-content 0 1fr 0;
+    `,
+    badge: css`
+      padding: 4px 3px;
+    `,
+    heading: css`
+      font-weight: 400;
+
+      > * {
```

Comment:
No * selectors allowed! I can't believe we don't have an eslint rule for this. Specify! Wildcards are inefficient

Reply:
I replaced with the hardcoded html tag.
The problem with that is if in the future the grafana-ui decide to use an other html tag, it will break this usage.

Ideally grafana-ui's `Card.Heading` provide a way to customize styles without relying on inline css. That is a comon problem I saw